packages/lib/server-only/user/get-all-users.ts

```diff
@@ -0,0 +1,71 @@
+import { Prisma } from '@prisma/client';
```

Comment:
Sorry I just noticed this, replace these imports with `@documenso/prisma/client` 😄 ❤️

Reply:
Yeah, but it doesn't have the `validator` and `UserWhereInput`. I removed it and made some changes though, so it should be fine now. 🙏